kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiCommands.java

```diff
@@ -521,6 +522,13 @@ public String handleMultiLineCmd(String line) throws IOException {
             "Console reader not initialized. This could happen when there "
                 + "is a multi-line command using -e option and which requires further reading from console");
       }
+
+      if (beeLine.getOpts().getInitFiles() != null) {
+        beeLine.getConsoleReader().shutdown();
+        beeLine.setConsoleReader(
```

Comment:
Is it better to call `BeeLine#initializeConsoleReader` directly, this method has set some properties of `ConsoleReader`?

Reply:
Thanks for the review, I have modified the code